android/config.json

```diff
@@ -2,11 +2,7 @@
     "copyFiles": [
         "FlurryPlugin.java"
     ],
-    "libraries": [
-    ],
-    "jars": [
-        "FlurryAnalytics-5.5.0.jar"
-    ],
     "injectionXML": "manifest.xml",
-	"injectionXSL": "manifest.xsl"
+	"injectionXSL": "manifest.xsl",
+	 "injectionTealeafModuleGradleXML": "gradletealeaf.xml"
```

Comment:
Hi Alex, the indentation isn't correct here

Reply:
Pushed improved identation